### DISCUSSION TITLE
Which data are collected when using Authenticator.cc

### DISCUSSION THREAD

**Discussion by aliceengrand** [Authenticator-Extension/Authenticator#1163] [Q&A] [answered]
Hi, 
Do you have any idea of which information the Authenticator.cc can grab from us when using his extension ? Is it the visited website ? Is it our username to login (actually this is our email address) ? Other ?
Many thanks,
Alice

**Comment by mymindstorm** [comment, marked as answer]
We don't store user data unless it is given as part of a QR code or direct user input. Even then, it stays in your computer and is never sent to us.

For more info see https://addons.mozilla.org/en-US/firefox/addon/auth-helper/privacy/
